resources/views/footer.blade.php

```diff
@@ -0,0 +1,49 @@
+<footer class="footer">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-6 footer_logo">
+                <a href="index.html"><img src="{{asset('img/logo.png')}}" alt="logo"></a>
+                <p>
+                    Lorem ipsum dolor amet natum latine copiosa at quo, suas labore saperet has there any quote for write lorem percit latineu suas dummy.
+                </p>
+                <ul>
+                    <li><a href="https://www.facebook.com"><i class="fab fa-facebook-f"></i></a></li>
+                    <li><a href="https://twitter.com"><i class="fab fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com"><i class="fab fa-linkedin-in"></i></a></li>
+                    <li><a href="https://www.instagram.com"><i class="fab fa-instagram"></i></a></li>
+                </ul>
+            </div>
+
+            <div class="col-md-6 link_footer">
+                <h4>Quick Links</h4>
+                <ul>
+                    <li>
+                        <a href="{{'/'}}">Home</a>
```

Comment:
you are over complicated here `"/"` is enough, but more correct is to have named route here